jsco4180Sup/jsco4180.db

```diff
@@ -840,6 +558,8 @@ record(ao, "$(P)COMP:A:SP")
     field(VAL, "100")
     
     field(EGU, "%")
+    field(FLNK, "$(P)COMP:D:SP")
+    info(autosaveFields, "VAL")
     info(INTEREST, "HIGH")
```

Comment:
Duplicated interest field on this record

Reply:
Setpoints are not usually interesting - usually only `VAL` not `VAL:SP` are marked as interesting as the scientists would only put a block on `VAL`

Reply:
Consider archiving these setpoints to improve the amount of diagnostics we have if the device goes wrong

Reply:
Made these changes as requested.